classes/class-swsales-reports.php

```diff
@@ -28,6 +29,36 @@ public static function add_reports_page() {
 		);
 	}
 
+	/**
+	 * Handles the Sales Export
+	 */
+	public static function sws_stats_csv() {
```

Comment:
Actually doesn't need the prefix since this is a method in a class.

Reply:
So we should rename to just stats_csv?

Reply:
Removed prefix in class method; updated prefix for ajax action.